src/modules/pages/ApiListPage/ApiListPage.js

```diff
@@ -18,7 +18,7 @@ const ApiListPage = () => (
                     <Row>
                         <SearchingContainer />
                         <Link to="/new">
-                            <Button label="Create New Api" />
+                            <Button mod="primary" label="Create New Api">+ Create New API</Button>
```

Comment:
yes, that's the stuff that should be removed, totally agree. I'll do it in separate PR 👍